sentry-android-distribution/build.gradle.kts

```diff
@@ -25,5 +25,8 @@ androidComponents.beforeVariants {
 
 dependencies {
   implementation(projects.sentry)
+  implementation(
```

Comment:
Other modules force the version (like `sentry-android-core`) which could cause unexpected behavior in consumers.

I thought using `implementation` is simpler to understand than a version constraint.

Reply:
hm I guess this is a bit weird that we leak the annotations to the runtime classpath (given that we only need them for compilation), but if there's no easy way around it, it's fine I guess.

Reply:
Annotations are quite leanient in the jvm world IIRC. If their dependencies aren’t present at compile time, it will still work so clients of the library can use an older version and it should be fine.